cypress/integration/adminCanCreateArticle.feature.js

```diff
@@ -24,6 +24,7 @@ describe('Admin is able to create an article ', () => {
         it('is expected to show article creation page', () => {
           const filepath = '../fixtures/imageFixture.png'
           cy.get('[data-cy=create-btn]').click()
+          cy.get('[data-cy=case-study-checkbox]').click()
```

Comment:
We probably need a test case for creating a regular article and a case_study both.

Reply:
I don't see a point.. there is no difference between them apart from one click on checkbox